### PR TITLE
Add support for ReliOn Premier meters (UPLOAD-373)

### PR DESCRIPTION
**Pull request by gniezen**
Addresses [UPLOAD-373] by adding support for:

- ReliOn Premier BLU
- ReliOn Premier Classic
- ReliOn Premier Voice

[UPLOAD-373]: https://tidepool.atlassian.net/browse/UPLOAD-373

### CODE REVIEW

File: lib/drivers/i-sens/careSens.js [hunk 1]
```diff
@@ -172,13 +327,142 @@ class CareSens {
...
+    const packet = {
+      bytes,
+      valid: false,
+      packet_len: 0,
+      payload: null,
+    };
+
+    const packetLength = bytes.length;
+    packet.packet_len = packetLength;
+    packet.valid = true;
```
Comment (krystophv) [inline lib/drivers/i-sens/careSens.js:348]: Not strictly necessary to change, but couldn't these assignments be done in the initial definition at 339?

File: lib/drivers/i-sens/careSens.js [hunk 2]
```diff
@@ -172,13 +327,142 @@ class CareSens {
...
+
+    const packet = CareSens.extractPacket(buffer.bytes());
+    if (packet.packet_len !== 0) {
+      // cleanup the buffer data
+      buffer.discard(packet.packet_len);
+    }
+
+    if (packet.valid) {
+      return packet;
+    }
```
Comment (krystophv) [inline lib/drivers/i-sens/careSens.js:366]: It seems that `CareSens.extractPacket()` will _always_ return a packet with the `.valid` set to `true`. If that's the case, then we expect this logic branch to _always_ be run, correct?

**Review by krystophv — DISMISSED** [review]
Couple comments/suggestions, but nothing that precludes this from approval. LGTM 👍

File: lib/drivers/i-sens/careSens.js (hunk 2 above)
Comment (gniezen) [reply lib/drivers/i-sens/careSens.js:366]: Yes! I guess that's what happens if you copy & paste a function from another driver and then end up changing it so much that it's almost not necessary anymore.

File: lib/drivers/i-sens/careSens.js (hunk 1 above)
Comment (gniezen) [reply lib/drivers/i-sens/careSens.js:348]: :+1:

**Comment by gniezen** [comment]
@krystophv Can you just check if I addressed your comments correctly?

**Review by krystophv — APPROVED** [review]
LGTM 👍

**Comment by gniezen** [comment]
@krystophv Sorry, just one last approval needed so that I can merge to master.

**Review by krystophv — APPROVED** [review]
LGTM 👍🏼

**Pull request closed without merging**